.gitignore

```diff
@@ -5,3 +5,4 @@ target/
 target-docker
 .vscode/tasks.json
 .idea/
+VERSION
```

Comment:
This make it not check-in the file at the top.

Reply:
git is smart enough to continue tracking changes to files that match .gitignore rules if those files are already committed in the repository